src/nvim/api/ui.c

```diff
@@ -128,7 +128,7 @@ void remote_ui_wait_for_attach(bool only_stdio)
 /// @param height  Requested screen rows
 /// @param options  |ui-option| map
 /// @param[out] err Error details, if any
-void nvim_ui_attach(uint64_t channel_id, Integer width, Integer height, Dict options, Error *err)
+void nvim_ui_attach(uint64_t channel_id, Integer width, Integer height, Integer pixel_width, Integer pixel_height, Dict options, Error *err)
```

Comment:
These added fields should be part of "options" dict

Reply:
Thanks for the review!

Just to clarify why I didn't yet react to this: I expect the type of these to change anyways to something representing optional values based on #31903, for which I think one option is to just put the optionals in a dict anyway.